mac

```diff
@@ -131,31 +82,40 @@ fi
 fancy_echo "Updating Homebrew formulas ..."
 brew update
 
-brew_install_or_upgrade 'zsh'
-brew_install_or_upgrade 'git'
-brew_install_or_upgrade 'postgres'
-brew_launchctl_restart 'postgresql'
-brew_install_or_upgrade 'redis'
-brew_launchctl_restart 'redis'
-brew_install_or_upgrade 'the_silver_searcher'
-brew_install_or_upgrade 'vim'
-brew_install_or_upgrade 'ctags'
-brew_install_or_upgrade 'tmux'
-brew_install_or_upgrade 'reattach-to-user-namespace'
-brew_install_or_upgrade 'imagemagick'
-brew_install_or_upgrade 'qt'
-brew_install_or_upgrade 'hub'
-brew_install_or_upgrade 'node'
-
-brew_install_or_upgrade 'rbenv'
-brew_install_or_upgrade 'ruby-build'
-
 # shellcheck disable=SC2016
 append_to_zshrc 'eval "$(rbenv init - --no-rehash zsh)"' 1
 
-brew_install_or_upgrade 'openssl'
-brew unlink openssl && brew link openssl --force
-brew_install_or_upgrade 'libyaml'
+BREWFILE=$(cat <<'EOF'
+tap 'thoughtbot/formulae'
+tap 'homebrew/services'
+brew 'zsh'
+brew 'git'
+brew 'postgres'
+brew 'redis'
+brew 'the_silver_searcher'
+brew 'vim'
+brew 'ctags'
+brew 'tmux'
+brew 'reattach-to-user-namespace'
+brew 'imagemagick'
+brew 'qt'
+brew 'hub'
+brew 'node'
+brew 'rbenv'
+brew 'ruby-build'
+brew 'libyaml'
+brew 'heroku-toolbelt'
+brew 'rcm'
+EOF
+)
+
+fancy_echo "Installing Homebrew Packages ..."
+  brew tap 'homebrew/bundle'
+  echo "$BREWFILE" | brew bundle -v --file=/dev/stdin
+  brew unlink openssl && brew link openssl --force
```

Comment:
Thanks for the comment! I agree and don't use this on my personal system. I meant for this PR to be exactly the current functionality.